client/components/NavigationBar.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { PropTypes } from 'prop-types';
+import { logout } from '../actions/auth/authActions';
+
+/**
+ * @param { NavigationBar } NavigationBar
+ * @returns { NavigationBar } NavigationBar
+ */
+class NavigationBar extends React.Component {
+  constructor(props) {
+    super(props);
+    this.logout = this.logout.bind(this);
+  }
+
+  logout(event) {
+    event.preventDefault();
+    this.props.logout();
+  }
+  /**
+   *
+   * @returns { Jsx } Jsx
+   */
+
+  loggedInMenu() {
+    return (
+      <ul className="nav nav-pills flex-column flex-sm-row flex-sm-fill text-sm-center topnav">
+        <li>
+          <Link className="nav-link" to="/recipes">
+            <i className="fa fa-home" aria-hidden="true">
+              {' '}
```

Comment:
I'm assuming the purpose of this is to create some form of padding for the content. You can achieve this with styling. Padding top or left will save you this.